app/contacts/contact_websocket.py

```diff
@@ -15,11 +18,16 @@ def __init__(self, services):
         self.stop_future = asyncio.Future()
 
     async def start(self):
-        web_socket = self.get_config('app.contact.websocket')
+        websocket_config = self.get_config('app.contact.websocket')
         try:
-            async with websockets.serve(self.handler.handle, *web_socket.split(':'), logger=self.log):
-                await self.stop_future
+            host, port = websocket_config.split(':')
```

Comment:
Can we also convert the `port` string to `int` in the validation check here, since an invalid conversion will also throw a `ValueError`?. Then we can avoid the potential exception throw when running `websockets.serve`